lib/report/lcovonly.js

```diff
@@ -60,8 +60,10 @@ Report.mix(LcovOnlyReport, {
 
         Object.keys(functions).forEach(function (key) {
             var stats = functions[key],
-                meta = functionMap[key];
-            writer.println('FNDA:' + [ stats, meta.name ].join(','));
+                meta = functionMap[key],
+                covered = stats > 0,
+                skipped = meta.skip;
+            writer.println('FNDA:' + [ (covered || !skipped) ? stats : 1, meta.name ].join(','));
```

Comment:
quite hare to read, maybe create a var on previous line for (covered || !skipped) ? stats : 1